### PR TITLE
Add helmet docs to guide

### PR DESCRIPTION
**Pull request by lhorie**

### CODE REVIEW

**Comment by KevinGrandon** [comment]
Deploy preview ready.

Built with commit 8206e98260536d1335bbe4d907a0984f3ef7b259

https://deploy-preview-15--fusionjs.netlify.com

File: content/modifying-html-template.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+Add the `Helmet` component to your React tree and add HTML elements inside of it to add/modify the respective HTML tag.
+
+```js
+import {Helmet} from 'fusion-plugin-react-helmet-async';
+
+const Root = () => (
+  <div>
+    <Helmet>
+      <title>Hello World</title>
+      <link rel="canonical" href="https://www.tacobell.com/" />
```
Comment (KevinGrandon) [inline content/modifying-html-template.md]: Might be better to point to something like fusionjs.com? :)

Comment (lhorie) [reply content/modifying-html-template.md]: lol why not :)

**Comment by KevinGrandon** [comment]
Looks good! I'm going to go ahead and press the button to land this because we don't have CI configured for this repo so the blocking is wrong here. Thanks for the update!

**Pull request merged**